sktime/forecasting/stream/_update.py

```diff
@@ -69,7 +62,8 @@ def __init__(
 
         super().__init__()
 
-        self.clone_tags(forecaster, TAGS_TO_CLONE)
+        self._set_delegated_tags(self.forecaster_)
+        self.set_tags(**{"fit_is_empty": False})
```

Comment:
For my understanding, is this needed here or can it be removed since it's already present in `_tags`?
```suggestion
```

Reply:
same as above, I think - `_set_delegated_tags` overides `_tags`, including `fit_is_empty`.